packages/fxa-auth-server/lib/routes/support.js

```diff
@@ -79,6 +80,7 @@ module.exports = (log, db, config, customs, zendeskClient) => {
             email,
             name: email,
           },
+          location,
```

Comment:
The Zendesk API accepts this and stores it somewhere user visible?

Reply:
There's discussion regarding the new Zendesk field over at https://github.com/mozilla/fxa/issues/2675#issuecomment-566730022.

Reply:
@jackiemunroe helped with the custom fields on the Zendesk side, and changes have been made: https://github.com/mozilla/fxa/pull/3711/files#diff-c01ed3e5dd5108a3df8ed8fc6bed2383R89-R94